backlog.py

```diff
@@ -0,0 +1,19 @@
+class Backlog:
```

Comment:
/pilot Please add docstrings to the class and create some unit tests (separate file)